generate-stackbrew-library.sh

```diff
@@ -26,6 +26,7 @@ fileCommit() {
 echo "# this file is generated via ${url}/blob/$(fileCommit "$self")/$self"
 echo
 echo "Maintainers: The Node.js Docker Team <${url}> (@nodejs)"
+echo "PR Bot Maintainer: Hank Brekke <brekkehj@hnryjms.io> (@hnryjms)"
```

Comment:
The generated output of this file is parsed in the official images, including the `Maintainers: ...` line, so this line will break the parser with an unknown field `PR Bot Maintainer`.  If necessary it could be generated as a comment `#`.  I guess it depends on what the docker-node maintainers want to do.